test/Array/rlexe.xml

```diff
@@ -239,6 +239,11 @@
       <baseline>pop3.baseline</baseline>
     </default>
   </test>
+	<test>
```

Comment:
Please can you match the indentation style in this file, I think it's 2 spaces per indent